consensus/scp/src/quorum_set.rs

```diff
@@ -28,29 +51,22 @@ pub struct QuorumSet<ID: GenericNodeId = NodeID> {
     pub members: Vec<QuorumSetMember<ID>>,
 }
 
-/// A member in a QuorumSet. Can be either a Node or another QuorumSet.
-#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash, Digestible)]
-#[serde(tag = "type", content = "args")]
-pub enum QuorumSetMember<ID: GenericNodeId> {
-    /// A single trusted entity with an identity.
-    Node(ID),
-
-    /// A quorum set can also be a member of a quorum set.
-    InnerSet(QuorumSet<ID>),
+impl<ID: GenericNodeId> PartialEq for QuorumSet<ID> {
+    fn eq(&self, other: &QuorumSet<ID>) -> bool {
+        if self.threshold == other.threshold && self.members.len() == other.members.len() {
+            // sort before comparing
```

Comment:
This is useful - another option would be to make `Vec<>` into `HashSet<>` - but that would be a larger change.